tests/test_sigalrm.c

```diff
@@ -9,24 +9,15 @@
 #include <unistd.h>
 #include <pthread.h>
 
-void alarm_handler(int dummy)
-{
-	printf("Received alarm!\n");
-#ifdef REPORT_RESULT
-	REPORT_RESULT(0);
-#endif
-	exit(0);
+void alarm_handler(int dummy) {
+  printf("Received alarm!\n");
+  exit(0);
 }
 
-int main()
-{
-	if (signal(SIGALRM, alarm_handler) == SIG_ERR)
-	{
-		printf("Error in signal()!\n");
-#ifdef REPORT_RESULT
-		REPORT_RESULT(1);
```

Comment:
What does `REPORT_RESULT` do and how does using `self.btest_exit` allows us to omit it?

This looks like `btest_exit` adds `exit:` to the expected string, right?
https://github.com/emscripten-core/emscripten/blob/4a9d5ab94f66b629b99c1104663bedbc80f799c1/tests/runner.py#L1591

But `REPORT_RESULT` macro prints `result: `?
https://github.com/emscripten-core/emscripten/blob/4a9d5ab94f66b629b99c1104663bedbc80f799c1/tests/report_result.h#L44-L48

Maybe I'm comparing incorrect things.